File: boj/11005.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int n = Integer.parseInt(st.nextToken());
        int b = Integer.parseInt(st.nextToken());
        StringBuilder ans = new StringBuilder();
        while (n != 0) {
            int temp = n % b;
            if (temp < 10) ans.append((char)(temp + 48));
            else ans.append((char)(temp + 55));
            n /= b;
        }
        ans.reverse();
        System.out.println(ans);
    }
}

File: boj/11005.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main(void) {
    ios_base::sync_with_stdio(false);
    int n, b; cin >> n >> b;
    string ans;
    while (n != 0) {
        int temp = n % b;
        if (temp < 10) ans += (temp + 48);
        else ans += (temp + 55);
        n /= b;
    }
    reverse(ans.begin(), ans.end());
    cout << ans << "\n";
    return 0;
}
